graderx/graders/import_submissions.py

```diff
@@ -15,11 +15,15 @@
 import base64
 from validator_collection import validators, checkers
 import urllib.parse
+from datetime import datetime
 
 
 def get_id_from_url(url):
     return re.findall(r'[-\w]{25,}', url)[0]
 
+def current_timestamp():
+    now = datetime.now()
+    return now.strftime("%Y%m%d%H%M%S%f")
```

Comment:
is this code duplicated in `graderx/graders/stdout_graders/c/lib/utils.py`?

Reply:
Fixed